alvr/server/cpp/tools/vulkan-layer/layer/alvr_x86_64.json

```diff
@@ -0,0 +1,45 @@
+{
+	"file_format_version" : "1.0.0",
+	"layer" : {
+		"name": "VK_LAYER_ALVR_capture",
+		"type": "GLOBAL",
+		"library_path": "./libVkLayer_window_system_integration.so",
+		"api_version": "1.0.68",
+		"implementation_version": "1",
+		"description": "ALVR display intercept layer",
+		"enable_environment": {
+			"SteamAppId":"250820"
+		},
+		"functions": {
+			"vkGetInstanceProcAddr": "wsi_layer_vkGetInstanceProcAddr",
+			"vkGetDeviceProcAddr": "wsi_layer_vkGetDeviceProcAddr"
+		},
+		"instance_extensions": [
```

Comment:
we should list the 2 more extensions provided by the layer
VK_EXT_acquire_xlib_display
VK_KHR_display